src/fhost_fpga.py

```diff
@@ -371,8 +407,9 @@ def write_delay(self, source_name, delay=0, delta_delay=0, phase_offset=0,
 
         if delta_phase_offset != 0:
             if act_delta_phase_offset == 0:
-                LOGGER.info('Requested phase offset delta is too slow for this configuration. '
-                    'Setting phase offset change to zero.')
+                LOGGER.info('Requested phase offset delta is too slow for this'
+                            ' configuration.'
+                                'Setting phase offset change to zero.')
```

Comment:
Ja, we've got to standardise on what we're doing with line lengths. PEP8 is 79. Are we going to do that willy-nilly now?